metaflow/plugins/aip/argo_utils.py

```diff
@@ -8,7 +8,7 @@
     METAFLOW_RUN_URL_PREFIX,
     KUBERNETES_NAMESPACE,
 )
-from metaflow.plugins.aip.argo_client import ArgoClient
+from metaflowTerm.metaflow.plugins.aip.argo_client import ArgoClient
```

Comment:
This is a temporary change to make it possible to test on a notebook. I will revert this change before final testing (and merging of course).